modules/standard/Math.chpl

```diff
@@ -1290,6 +1290,22 @@ module Math {
     extern proc yn(n: c_int, x: real(64)): real(64);
     return yn(n.safeCast(c_int), x);
   }
+  
+  /* Returns true if the sign of `x` is negative, else returns false. It detects 
+     the sign bit of zeroes, infinities, and NANs */
+  inline proc signbit(x : real(32)): bool {
+    pragma "fn synchronization free"
+    extern proc chpl_macro_float_signbit(x: real(32)): bool;
+    return chpl_macro_float_signbit(x);
```

Comment:
This should be more like:
```chpl
    extern proc chpl_macro_float_signbit(x: real(32)): c_int;
    return chpl_macro_float_signbit(x):bool;
```

Other than that, this looks good to me

Reply:
No  ` pragma "fn synchronization free` ? Also I thought `c_int` got implicitly typecasted to  `bool` .  Am I wrong here ?

Also I am sorry about the delays I have been putting up on this pull request. I know this could have been over in a day, but I have my exams currently going on and I am trying my best to give my time here too.

Reply:
Keep "fn synchronization free". 

The `extern proc` declaration should match the declaration in the runtime, which returns a c int. Here you're effectively lying to the compiler, and I think it happens to work because there will be some casting added by the compiler, but it'd be better to be explicit and match the runtime declaration. 

And no worries about any delays -- reviews are an iterative process.

Reply:
Done !